app/src/main/kotlin/com/bitwarden/authenticator/data/platform/repository/SettingsRepositoryImpl.kt

```diff
@@ -43,6 +43,8 @@ class SettingsRepositoryImpl(
     override var authenticatorAlertThresholdSeconds = settingsDiskSource.getAlertThresholdSeconds()
 
     override var defaultSaveOption: DefaultSaveOption by settingsDiskSource::defaultSaveOption
+    override val defaultSaveOptionFlow: Flow<DefaultSaveOption>
```

Comment:
Can ad a newline here?
```suggestion

    override val defaultSaveOptionFlow: Flow<DefaultSaveOption>
```

Reply:
Thanks :)